src/main/java/com/google/maps/model/PlaceDetails.java

```diff
@@ -0,0 +1,197 @@
+         */
+        UNKNOWN
+      }
+
+      /** type is the name of the aspect that is being rated. */
+      public RatingType type;
+
+      /** rating is the user's rating for this particular aspect, from 0 to 3. */
+      public int rating;
+    }
+
+    /** aspects contains a collection of AspectRating objects, each of which provides a rating of
+     * a single attribute of the establishment. */
+    public AspectRating[] aspects;
+
+    /** authorName the name of the user who submitted the review. Anonymous reviews are attributed
+     * to "A Google user". */
+    public String authorName;
+
+    /** authorUrl the URL to the users Google+ profile, if available. */
+    public URL authorUrl;
+
+    /** language an IETF language code indicating the language used in the user's review. */
+    public String language;
+
+    /** rating the user's overall rating for this place. This is a whole number, ranging from 1 to
+     * 5. */
+    public int rating;
+
+    /** text is the user's review. When reviewing a location with Google Places, text reviews are
+     * considered optional. */
+    public String text;
+
+    /** time is the time that the review was submitted, as seconds since epoch.*/
+    public long time; // TODO(brettmorgan): figure out how to convert this to a DateTime.
+  }
+
+  /** reviews is an array of up to five reviews. */
+  public Review[] reviews;
+
+  /** types contains an array of feature types describing the given result. */
+  public String[] types;
+
+  /** url contains the URL of the official Google page for this place. This will be the
+   * establishment's Google+ page if the Google+ page exists, otherwise it will be the
+   * Google-owned page that contains the best available information about the place. Applications
+   * must link to or embed this page on any screen that shows detailed results about the place to
+   * the user. */
+  public URL url;
+
+  /** utcOffset contains the number of minutes this place’s current timezone is offset from UTC. */
+  public int utcOffset;
+
+  /** vicinity lists a simplified address for the place, including the street name, street number,
+   * and locality, but not the province/state, postal code, or country. */
+  public String vicinity;
+
+  /** website lists the authoritative website for this place, such as a business' homepage. */
+  public URL website;
+
+  /** htmlAttributions contains an array of attributions about this listing which must be displayed
+   * to the user. */
+  public String[] htmlAttributions;
+
+  /** id is deprecated. Please do not use. This field may stop being returned at any time.
+   *
+   * @deprecated This field is deprecated. See the deprecation warning at the top of
+   * <a href="https://developers.google.com/places/web-service/details">this page</a>
+   */
+  public String id;
+
+  /** reference is deprecated. Please do not use. This field may stop being returned at any time.
```

Comment:
I'd vote to leave it out. We're in the fortunate case where we are building a new library so we can leave out the legacy we don't want to support. If it becomes a problem we can always add it in later.

Ditto for anything that's being introduced as @deprecated.